src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    &:hover {
+      color: $roo;
+      transition: 0.3s;
```

Comment:
The transition property should be set for the normal state of the element, not inside the hover effect